Makefile

```
@@ -59,14 +59,18 @@ build-e2e:
 
 .PHONY: test-e2e
 test-e2e: ## Run openshift specific e2e test
+	# Run Disruptive tests first. Those might include things causing all masters to restart, and should run serially
+
+	hack/ci-integration.sh $(GINKGO_ARGS) -focus="Disruptive" || (hack/junitmerge.sh && exit 1)
```

Should this use the same focus as the others skip?
```suggestion
	hack/ci-integration.sh $(GINKGO_ARGS) -focus="\[Disruptive\]" || (hack/junitmerge.sh && exit 1)
```

---

Fixed, PTAL.